app/views/layouts/shared/_navbar.html.erb

```diff
@@ -19,75 +19,39 @@
-              <% if current_user.adopter_foster_account %>
-                <li>
-                  <%= link_to adopter_foster_dashboard_index_path, class: 'dropdown-item' do %>
-                    <i class="fe fe-user me-2"></i> Dashboard
-                  <% end %>
-                </li>
-              <% end %>
-              <% if current_user.staff_account && !current_user.staff_account.deactivated? %>
-                <li>
-                  <%= link_to dashboard_index_path, class: 'dropdown-item' do %>
-                    <i class="fe fe-star me-2"></i> Dashboard
-                  <% end %>
-                </li>
-              <% end %>
-              <li>
-                <%= link_to edit_user_registration_path, class: 'dropdown-item' do %>
-                  <i class="fe fe-settings me-2"></i> Account Settings
-                <% end %>
-              </li>
-              <div class="dropdown-divider"></div>
-              <li>
-                <%= button_to destroy_user_session_path, method: :delete, data: { turbo: false }, class: 'dropdown-item' do %>
-                  <i class="fe fe-power me-2"></i> Log Out
-                <% end %>
-              </li>
-            </ul>
-            <% if current_user.staff_account && current_user.has_role?(:admin, current_user.staff_account.organization) %>
-              <div class="ms-4">*Admin*</div>
-            <% end %>
-          <% end %>
-        </div>
-      </div>
+      <% if user_signed_in? %>
+        <li class="nav-item d-block">
+          <%= button_to 'Log Out', destroy_user_session_path, method: :delete, data: { turbo: false }, class: 'ms-2 btn btn-primary' %>
+        </li>
+      <% end %>
     </div>
 
     <!-- Collapse -->
     <div class="justify-content-end collapse navbar-collapse" id="navbar-default3">
       <ul class="navbar-nav">
-
-        <li class='nav-item'>
-          <%= active_link_to 'Adopt', new_user_registration_path, class: 'nav-link' %>
-        </li>
         <li class='nav-item'>
-          <%= active_link_to 'Get Involved', '#', class: 'nav-link' %>
+          <%= active_link_to 'Pets', adoptable_pets_path, class: 'nav-link' %>
         </li>
         <li class='nav-item'>
-          <%= active_link_to 'About Us', about_us_path, class: 'nav-link' %>
+          <%= active_link_to 'About', about_us_path, class: 'nav-link' %>
         </li>
         <li class='nav-item'>
-          <%= active_link_to 'Partners', partners_path, class: 'nav-link' %>
+          <%= active_link_to 'Contact', new_contact_path, class: 'nav-link' %>
         </li>
         <li class="nav-item">
           <%= active_link_to 'Donate', donate_path, class: 'nav-link' %>
         </li>
         <% if user_signed_in? %>
-          <li class="nav-item d-block d-lg-none">
-            <%= button_to 'Log Out', destroy_user_session_path, method: :delete, data: { turbo: false }, class: 'nav-link' %>
-          </li>
+          <% if current_user.adopter_foster_account %>
```

Comment:
See https://github.com/rubyforgood/pet-rescue/pull/642/files#r1564502741

In this case it would be:
```ruby
<% if allowed_to?(:index?, with: Organizations::AdopterFosterDashboardPolicy, context: {organization: Current.organization}) %>
```

Reply:
Ah yes good call. I made these two changes!